.github/actions/setup-rust/action.yml

```diff
@@ -0,0 +1,42 @@
+name: "Setup Rust"
```

Comment:
you can pull this actions from our `actions` repo

Reply:
Good, idea, but these are different. Shall we make a follow up issue on `actions`?

Reply:
https://github.com/spiraldb/actions/issues/10